service/src/main/java/bio/terra/cbas/dependencies/wes/CromwellService.java

```diff
@@ -150,9 +154,7 @@ public String buildWorkflowOptionsJson(Boolean isCallCachingEnabled) {
 
     // Path for cromwell to write workflow logs to.
     Optional<String> finalWorkflowLogDir = cromwellClient.getFinalWorkflowLogDirOption();
-    if (finalWorkflowLogDir.isPresent()) {
-      workflowOptions.put("final_workflow_log_dir", finalWorkflowLogDir.get());
-    }
+    finalWorkflowLogDir.ifPresent(s -> workflowOptions.put("final_workflow_log_dir", s));
```

Comment:
I like this change, but it seems slightly odd to see it happening in the middle of this PR? 😆

Reply:
Not quite sure where this change actually came from myself!

Reply:
Looking at the changelog, this was the result of `spotlessApply`